LoopKit/TherapySettings.swift

```diff
@@ -44,7 +44,7 @@ public struct TherapySettings: Equatable, Codable {
         insulinSensitivitySchedule: InsulinSensitivitySchedule? = nil,
         carbRatioSchedule: CarbRatioSchedule? = nil,
         basalRateSchedule: BasalRateSchedule? = nil,
-        insulinModel: StoredSettings.InsulinModel? = nil
+        insulinModel: InsulinModelSettings? = nil
```

Comment:
[nit] consider naming this `insulinModelSettings`, to make the distinction clear